README.md

```diff
@@ -12,7 +12,7 @@ classfiles.
 Class files are rendered with ASM's [`Textifier`](https://asm.ow2.io/javadoc/org/objectweb/asm/util/Textifier.html)
 and with `scalap`. Other files are rendered as-is.
```

Comment:
In the neighborhood, the hyphen is not desirable here. "The house was sold as is." But "on an as-is basis." I think this is different from "ass-backwards".